AAT_raspberry/storage.py

```diff
@@ -0,0 +1,20 @@
+filename = 'store'
+
+# inserts val to end of store file
+def insert(val):
+  with open(filename, 'a') as f:
+    f.write(val + '\n')
+
+# gets all stored values
+def get():
+  with open(filename, 'r') as f:
+    return map(str.strip, f.readlines())
+
+# deletes first numLines number of values
+def delete(numLines):
+  lines = []
+  with open(filename, 'r') as f:
+    lines = f.readlines()
+
+  with open(filename, 'w') as f:
+    f.writelines(lines[numLines:])
```

Comment:
The "with" operation will handle opening and closing the file once execution leaves its block. For instance, in the delete method, f will be opened on line 16, and then closed after line 17. Same goes for lines 19 and 20. 

Also, opening with the "w" flag will clear the old content of the file before the write.